emottak-utils/src/main/kotlin/no/nav/emottak/utils/kafka/KafkaPublisherClient.kt

```diff
@@ -1,57 +1,21 @@
 package no.nav.emottak.utils.kafka
 
+import io.github.nomisRev.kafka.publisher.KafkaPublisher
 import no.nav.emottak.utils.config.Kafka
-import no.nav.emottak.utils.config.toProperties
-import org.apache.kafka.clients.producer.KafkaProducer
-import org.apache.kafka.clients.producer.ProducerConfig
+import no.nav.emottak.utils.config.toKafkaPublisherSettings
 import org.apache.kafka.clients.producer.ProducerRecord
-import org.apache.kafka.common.serialization.ByteArraySerializer
-import org.apache.kafka.common.serialization.StringSerializer
+import org.apache.kafka.clients.producer.RecordMetadata
 
-class KafkaPublisherClient(val topic: String, val config: Kafka) : AutoCloseable {
-    // private val log = LoggerFactory.getLogger(this.javaClass)
+class KafkaPublisherClient(
```

Comment:
https://github.com/nomisRev/kotlin-kafka

Dette biblioteket ser ut til å være relativt lite og ikke så kjent, med få følgere og bare 2 maintainere. 
Hvis hovedformålet med å introdusere det er å redusere boilerplate-kode vi allerede har på plass her, er jeg usikker på om det gir nok verdi til å forsvare innføringen av den. Men dersom det har større fordeler som kan gi betydelig merverdi, er jeg åpen for videre diskusjon.

Reply:
Vi buker det i hele smtp-transport og nesten alle steder i ebxml-prosessor....

Reply:
@ivanskodje Se siste svaret mitt til @OleksandrChmyrNAV. Jeg har presentert dette biblioteket på delingstimen tidligere og føler at jeg gjentar meg selv litt her 😅

Reply:
Det ble vedtatt på Fredag å bruke denne:
https://nav-it.slack.com/archives/C02E5M1N8A2/p1742552723465319